projects/packages/waf/src/class-waf-runner.php

```diff
@@ -39,6 +40,18 @@ public static function define_mode() {
 		}
 	}
 
+	/**
+	 * Set the mode definition if it has not been set.
+	 *
+	 * @return void
+	 */
+	public static function define_share_data() {
+		if ( ! defined( 'JETPACK_WAF_SHARE_DATA' ) ) {
+			$share_data_option = get_option( self::SHARE_DATA_OPTION_NAME, false );
```

Comment:
Can we update these defaults to true based on 0-as-1202204517249591/1202211719604597/f ?

Reply:
Yes, but I rather do it in the hooks that initialize the module, if we just change the default to true here we'll have the weird behavior of the option not existing at all and being toggled off in the UI and then the logs being written. I was thinking of just setting the option as true when the module is enabled so it shows as enabled in the UI as well.

Reply:
K.